server.js

```diff
@@ -6038,6 +6039,8 @@ function versionColor(version) {
   var first = version[0];
   if (first === 'v') {
     first = version[1];
+  } else if (/^version\/[0-9]/.test(version)) {
+    version = 'v' + version.substring(8);
```

Comment:
Good point, I forgot this was in here. I'll move it to a different branch. The approach of finding the first digit works too, but it depends how restrictive we want/need the regex to be. Something like `/^[^0-9]*(.+)+/` catches everything from `v0.1.0` to `imahoot1234`. That last one might not really fall into the (semantic) version category.

I'll open a new PR for this so we can continue the discussion.